.github/workflows/pull-request_term-server.yml

```diff
@@ -0,0 +1,50 @@
+name: PR Check - term-server
+
+on:
+  pull_request:
+    paths:
+      - "packages/terminology/**"
+  merge_group:
+  workflow_dispatch:
+
+jobs:
+  check-commits:
+    uses: ./.github/workflows/_check-commits.yml
+    secrets: inherit
+
+  # Not reusing _build because that one is focused on the monorepo and this package
+  # is a standalone one, with no dependencies on the other packages.
+  check-pr:
+    runs-on: ubuntu-latest
+    env:
+      working-directory: "./packages/terminology"
+    steps:
+      - name: Log Environment
+        run: |
+          env
+        shell: bash
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Setup NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.14"
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Log Git Ref
+        run: |
+          echo "Git ref: $(git rev-parse HEAD)"
+        shell: bash
+
+      - name: Install
+        run: |
+          pwd
+          npm install
+        working-directory: ${{env.working-directory}}
+
+      - name: Build (linters)
+        run: npm run build
+        working-directory: ${{env.working-directory}}
```

Comment:
Let's add a test step as well, even if the respective npm script just logs "no tests" for now, so the plumbing is in place, please.